extensions/ql-vscode/src/mocks/gh-api-request.ts

```diff
@@ -28,7 +32,7 @@ export interface SubmitVariantAnalysisRequest {
   },
   response: {
     status: number,
-    body: VariantAnalysis
+    body: VariantAnalysis | BasicErorResponse | undefined
   }
```

Comment:
I think it would have been really nice to use something like:

```ts
  response: {
    status: 201,
    body: VariantAnalysis
  } | {
    status: number,
    body?: BasicErorResponse
  }
```

However, this doesn't actually seem to be validated properly by #1642, so I don't think it's really worth it.